docs/manual/developer/06_contributing_with_content.md

```diff
@@ -1197,7 +1197,142 @@ See some of the existing testing code in `cmake/SSGCommon.cmake`.
 
 ### Unit tests
 
-TBD
+Running unit tests is very useful way to verify the content being
+written without requiring a full-blown OpenShift cluster.
+
+The unit testing approach for content requires a container
+where the scanning will be executed. This container will
+be a small replication of the environment where the content
+is intended to run.
+
+To build the container, from the project's root
+directory, execute the following command:
+
+    podman build --build-arg CLIENT_PUBLIC_KEY=(cat ~/.ssh/id_rsa.pub) -t ssg_test_suite -f Dockerfiles/test_suite-fedora
```

Comment:
Is this line correct? I think first you meant to write `$(cat ~/.ssh/id_rsa.pub)`. But either way, I'm getting:
```
Error: accepts at most 1 arg(s), received 2
```
btw what worked for me was to first set the contents of the pubkey into a variable and then run:
```
podman build -t ssg_test_suite  -f Dockerfiles/test_suite-fedora --build-arg CLIENT_PUBLIC_KEY="${CLIENT_PUBLIC_KEY}"
```

Reply:
oops! Sorry, this was `fish` format, not bash